Guide/src/dev_guide/tests/vmm.md

```diff
@@ -14,8 +14,16 @@ The OpenVMM repo contains a set of "heavyweight" VMM tests that fully boot a
 virtual machine and run validation against it. Unlike Unit tests, these are all
 centralized in a single top-level `vmm_tests` directory.
 
+The OpenVMM CI pipeline will run the full test suite; you'd typically run only
+the tests relevant to the changes you're working on.
+
 ### Running VMM Tests
 
+```admonish warning
+`cargo nextest run` won't rebuild any of your changes. Make sure you `cargo build`
+or `cargo xflowey igvm [RECIPE]` first!
```

Comment:
This isn't the best wording.

It will rebuild your _test_ changes, but it won't rebuild _artifacts_.